Include/pymacro.h

```diff
@@ -0,0 +1,100 @@
+#ifndef Py_PYMACRO_H
+#define Py_PYMACRO_H
+
+/* Minimum value between x and y */
+#define Py_MIN(x, y) (((x) > (y)) ? (y) : (x))
```

Comment:
These defines are supposed to eventually replace all the similar ones sprinkled around the code, right? (not requesting a PR change, just making a note that upstream seems to have done that and it's a good idea for a future PR)

```text
$ rg 'define MIN\('
Objects/frameobject.c
12:#define MIN(a, b) ((a) < (b) ? (a) : (b))

Objects/floatobject.c
15:#define MIN(x, y) ((x) < (y) ? (x) : (y))

Objects/longobject.c
32:#define MIN(x, y) ((x) > (y) ? (y) : (x))

Objects/stringlib/localeutil.h
9:#define MIN(x, y) ((x) < (y) ? (x) : (y))

Modules/bz2module.c
76:#define MIN(X, Y) (((X) < (Y)) ? (X) : (Y))

Modules/_cursesmodule.c
167:#define MIN(x,y) ((x) < (y) ? (x) : (y))

Modules/_multiprocessing/multiprocessing.h
185:#  define MIN(x, y) ((x) < (y) ? x : y)
```

Reply:
Ah yeah. I believe it to be so.